pandas/compat/_optional.py

```diff
@@ -16,13 +16,13 @@
     "openpyxl": "2.4.0",
     "pandas_gbq": "0.8.0",
     "pyarrow": "0.9.0",
-    "pytables": "3.4.2",
+    "tables": "3.4.2",
     "s3fs": "0.0.8",
     "scipy": "0.19.0",
     "sqlalchemy": "1.1.4",
     "xarray": "0.8.2",
     "xlrd": "1.0.0",
-    "xlwt": "2.4.0",
+    "xlwt": "1.0.0",
```

Comment:
Why is this being downgraded?

Reply:
I messed up in the previous PR. That was openpyxl's min version.